gnome-abrt.spec.in

```diff
@@ -73,7 +73,8 @@ desktop-file-install \
 
 
 %check
-make check
+# do not fail on pylint warnings
+make check || :
```

Comment:
The check was fatal because, in the history, I released some version that included errors.
Make check that is not fatal on is pure nonsense - you would do better, if you removed it.
Currently, it looks like that gnome-abrt can check itself, but it is a lie!
Comment saying that `|| :` means do not fail is not very helpful.
I would appreciate "why you believe it is a good idea to remove checks completely".

Reply:
BTW, if you want to run checks without pylint you can passs `--with-nopylint` to `%configure`.

Reply:
https://github.com/mhabrnal/gnome-abrt/commit/1c93c486e954d2723b607833c58567a014675f2f#diff-c3189e78ca44c8f42b1bd5a965e340c2